Makefile

```diff
@@ -0,0 +1,22 @@
+PROJ_NAME={{.Name}}
+JEKYLL_VERSION=3.5
+IMAGE_NAME=doc-test
```

Comment:
Is this image pushed to the repo? I'm not able to use `docker run gcr.io/heptio-images/doc-test:latest`

Reply:
I included the image making code.

Reply:
Awesome, thanks. I was able to get it to work, but had to check out the doc-site-gen repo in order to do so.

Reply:
Oh darn it. Good point.

Reply:
fwiw, I think this approach is fine